src/components/ds/button/button.tsx

```diff
@@ -0,0 +1,30 @@
+import styles from "./button.module.css";
+import { Loading } from "../../icons/loading";
+import classNames from "classnames";
+
+const Button: React.FC<{
+  handleClick?: () => void;
+  text: string;
```

Comment:
```suggestion
  children?: React.ReactNode;
```

lets use children instead of `text` so its more extensive. You now use it like: 

```jsx
<Button>Build</Button>
```